crates/common/src/backend.rs

```diff
@@ -0,0 +1,8 @@
+use sparse_merkle_tree::H256;
```

Comment:
The gw_jsonrpc_types::godwoken::BackendInfo is enough for use, we can remove this module

Reply:
This is due to two impl of H256.
gw_jsonrpc_types::godwoken uses ckb_fixed_hash::H256 only and the others use smt::H256. And the common crate is used by most crates, so I think this is a good place to make value to value transformations ?